src/components/App/App.css

```diff
@@ -6,16 +6,46 @@
 
 html {
   margin: 1.5em 2em 0 2em;
+  background-color: #191725
 }
 
 body {
   margin: auto;
   max-width: 1200px;
+  min-height: 100vh;
 }
 
 main {
-    display: flex;
-    justify-content: center;
```

Comment:
looks like this change disrupted the formatting, mainly on `/trending` but the placeholders on the tab pages are also no longer centered.  Why was this removed?
<img width="1335" alt="Screenshot 2023-07-27 at 7 13 22 PM" src="https://github.com/mbenfowler/sound-stash/assets/32660020/7529fbd0-4381-4cac-9157-a1617e210362">

Reply:
yeah, something went off here, but if should be an easy fix. I didn't focus on this since we have a ticket for the styling. Also, I don't think the tabs were ever perfectly centred, but it might shift when we add a login and user icon, so I didn't fix it.